docker/Dockerfile

```diff
@@ -0,0 +1,49 @@
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.5-240 as builder
```

Comment:
what about renaming this file `Dockerfile.appsre` or something similar?

Reply:
@r00ta sure, it makes sense to make the objective clear.